src/analysis/max_min_driving_force.jl

```diff
@@ -0,0 +1,163 @@
+"""
+    max_min_driving_force(
+        model::StandardModel,
+        thermodynamic_data::Dict{String, Float64},
+        optimizer;
+        modifications = [],
+        proton_id = "h_c",
+        water_id = "h2o_c",
+        concentration_ratios = [
+            ("atp_c", "adp_c", 10.0),
+            ("adp_c", "amp_c", 1.0),
+            ("nadph_c", "nadp_c", 10.0),
+            ("nadh_c", "nad_c", 0.1),
+        ],
+        constant_concentrations = [
+            ("coa_c", 1e-3),
+            ("co2_c", 10e-6),
+            ("pi_c", 10e-3),
+            ("ppi_c", 1e-3),
+        ],
+        concentration_lb = 1e-6,
+        concentration_ub = 10e-3,
+)
```

Comment:
sneaky

Reply:
will add units, but these bounds are also based on physiological observations + used in many papers